spec/spec_helper_acceptance.rb

```diff
@@ -164,7 +163,6 @@ def teardown_puppet_on(host)
 package { ['puppet-agent', 'puppet', 'mcollective', 'mcollective-client']: ensure => purged }
   EOS
   on host, puppet('apply', '-e', "\"#{pp}\"")
-  remove_defaults_on host
```

Comment:
Filing a BKR ticket about this, the remove_defaults_on code can't be called twice with `type: aio` set in the config file.